src/include/stir/listmode/CListModeDataECAT8_32bit.h

```diff
@@ -76,15 +76,13 @@ class CListModeDataECAT8_32bit : public CListModeData
   /*! \todo this might depend on the acquisition parameters */
   virtual bool has_delayeds() const { return true; }
 
-    virtual
-    shared_ptr<ProjDataInfo> get_proj_data_info_sptr() const;
 private:
   typedef CListRecordECAT8_32bit CListRecordT;
   std::string listmode_filename;
   shared_ptr<InputStreamWithRecords<CListRecordT, bool> > current_lm_data_ptr;
   //moved at CListModeData
 //  shared_ptr<ProjDataInfo> proj_data_info_sptr;
-  InterfileHeader interfile_parser;
+  InterfilePDFSHeader interfile_parser;
```

Comment:
why do we need a sinogram header? Won't it fail on the data? (maybe it works for now as it'll parse the segment stuff). If it's alright, please add a comment accordingly, marked with TODO (also put it in a `\todo` in the doxygen header).

Reply:
I changed this in a  `InterfileListmodeHeaderSiemens`. Is more appropriate I think